app/views/submissions/submit_comment.erb

```diff
@@ -1,14 +1,18 @@
-<form accept-charset="UTF-8" action="/submissions/<%= submission.key %>/nitpick" method="POST">
+<% if current_user.guest? %>
+  <p> You're not logged in right now.Please login via <a href="/login">GitHub </a> to comment </p>
```

Comment:
Hmmm.. It redirects you to your dashboard. I might have to see what this href="<%= login_url(return_path) %>"> is doing